pkg/exec/exec.go

```diff
@@ -64,3 +64,13 @@ func (exe *Executor) GoInstall(ctx context.Context, path, gobin string) (int, er
 	cmd.Env = append(os.Environ(), "GOBIN="+gobin)
 	return exe.exec(ctx, cmd)
 }
+
+func (exe *Executor) HdiutilDetach(ctx context.Context, mountPath string) (int, error) {
+	cmd := exe.command(exec.Command("hdiutil", "detach", mountPath, "-quiet"))
```

Comment:
I'm considering if we should set `-quiet` option or not.

`-quiet` option removes the error message.

```console
$ hdiutil attach  get.volta.sh
hdiutil: attach failed - image not recognized

$ hdiutil attach  get.volta.sh -quiet
$ echo $?
1
```

On the other hand, `-quiet` option makes the log clean.